native-vm-just-run-ninja/project.cpp

```diff
@@ -0,0 +1,250 @@
+#include <iostream>
```

Comment:
What is this file for?

Reply:
this is like the windows native-vm-vs-project but on linux, with ninja.